app/main.py

```diff
@@ -217,18 +217,31 @@ def select_charset(request):
 
 def select_lang(request, config=None):
     """Selects the best language to use for a given request.  The 'lang' query
-    parameter has priority, then the django_language cookie, then the first
-    language in the language menu, then the default setting."""
-    default_lang = (config and
-                    config.language_menu_options and
-                    config.language_menu_options[0])
+    parameter has priority, then the django_language cookie, then
```

Comment:
Looks like your pull request contains changes from other changes. Maybe try git pull origin master and git push again?

Reply:
I will do that.